NEWS.md

```diff
@@ -11,13 +11,22 @@ next ():
 
 master ():
 
+- optimized network settings for the new structure since the link
+  length fix : less peers for the same bandwidth should result in
+  higher throughput per connection. This allows for less powerful
+  devices to join (with low bandwidth settings) and should provide
+  better bandwidth utilization for very fast nodes.
+- Re-enable RSA-based ciphers
```

Comment:
Shouldn't changes in core security areas such as cryptography mention things such as
- why they were done
- which subsystems they affect
?